src/v3/test/integration/__snapshots__/authenticator-expiry-warning-password.test.tsx.snap

```diff
@@ -967,7 +967,7 @@ exports[`authenticator-expiry-warning-password should render form 1`] = `
                     data-se="o-form-explain"
                     id="generated"
                   >
-                    When your password expires, you will have to change your password before you can login to your Localhost account.
+                    L10N_ERROR[idx.password.expiring.message]
```

Comment:
not sure why im getting an L10N error here

Reply:
Same here, this translation was recently updated so you just need to run yarn build:release locally to make sure the login.json file is updated with the latest.